src/style.css

```diff
@@ -1 +1,30 @@
-/* add styles here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  align-items: stretch;
+  box-sizing: border-box;
+}
+
+.stars__star {
+  background-image: url(../stars/white_star.png);
+  background-repeat: no-repeat;
+  list-style-type: none;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.classes` and if needed with `:pseudo-classes`, `pseudo-elements` and `[attributes]`. For example, instead of `list-style-type: none;` on `.stars__star`, create a class that applies to the `ul` element that contains the `.stars__star` items.